src/backend/src/controllers/projects.controllers.ts

```diff
@@ -299,3 +299,54 @@ export const editProject = async (req: Request, res: Response) => {
   // return the updated work package
   return res.status(200).json(updatedProject);
 };
+
+export const setProjectTeam = async (req: Request, res: Response) => {
+  const { body } = req;
+
+  // check for valid WBS number
+  const parsedWbs: WbsNumber = validateWBS(req.params.wbsNum);
+
+  if (!isProject(parsedWbs)) {
+    return res.status(404).json({ message: `${req.params.wbsNum} is not a valid project WBS #!` });
+  }
+
+  const wbsEle = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsNumber: {
+        carNumber: parsedWbs.carNumber,
+        projectNumber: parsedWbs.projectNumber,
+        workPackageNumber: parsedWbs.workPackageNumber
+      }
+    },
+    include: {
+      project: true
+    }
+  });
+
+  if (wbsEle === null) {
+    return res.status(404).json({ message: `project ${req.params.wbsNum} not found!` });
```

Comment:
need to turn the wbsnum into a string otherwise it just returns as `[Object object]`

Reply:
![image](https://user-images.githubusercontent.com/17571655/206347839-46a0aa8e-d86c-4757-bf54-28087b9a1ddb.png)
This seems to work when tested from my side, and line [#310 ](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/472/files#diff-89f350e45ce637e60309e0a420e29e416518e86fa3b99ef5db0202a7115ad5a9R310) also does the same thing with no issue.

Reply:
huh ok nvm, i guess cuz it comes from params its a string